src/Amazon.Extensions.CognitoAuthentication/Util/CognitoAuthHelper.cs

```diff
@@ -40,9 +40,28 @@ internal static SHA256 Sha256
                 {
                     sha256 = SHA256.Create();
                 }
+
                 return sha256;
             }
         }
+    
+        internal static string AssemblyFileVersion
+        {
+            get
+            {
+                if (assemblyFileVersion == null)
+                {
+                    var assembly = typeof(CognitoAuthHelper).GetTypeInfo().Assembly;
+                    AssemblyFileVersionAttribute attribute = assembly.GetCustomAttribute(typeof(AssemblyFileVersionAttribute)) as AssemblyFileVersionAttribute;
+
+                    assemblyFileVersion = attribute == null ? "Unknown" : attribute.Version;
+                }
+                
+                return assemblyFileVersion;
+            }
+        }
+        
+        private const string AssemblyFileVersionCachePath = "Cognito.AssemblyFileVersion";
```

Comment:
I think this is orphan code from the previous revision. Can you delete this?

Reply:
Removed it